pisa/stages/osc/pi_osc_params.py

```diff
@@ -3,6 +3,10 @@
 """
 OscParams: Characterize neutrino oscillation parameters
            (mixing angles, Dirac-type CP-violating phase, mass splittings)
+
+changed by Elisa Lohfink (ellohfin; elohfink@icecube.wisc.edu) 
```

Comment:
You seem to have left the NSI coupling getters and setters in this class. Probably not on purpose?

Reply:
There's also some (empty) dummy unit testing functionality at the bottom of https://github.com/thehrh/pisa/blob/nsi_reparameterisation/pisa/stages/osc/pi_osc_params.py, which could be useful to keep so the `TODO` is at least documented.

Reply:
(Note that raising `NotImplementedError` will keep our whole-PISA-unit-test-runner-script from passing, so preferably that is turned into at least some tests, but if not, that should be just turned into a `pass` statement for now and e.g. add a `# TODO` comment or something.)

Reply:
Can't such an error be handled while testing?

Reply:
My bad. I know it prevents it from passing. I found that awareness of a lacking unit test should be escalated.

Reply:
@ellohfin : yes, we could add "allow the following errors to pass but turn them into warnings" or something logic. But that's a lot more work than I'm willing to put into it, and then we end up with a lot of weird stuff in the unit test runner script, and this kind of thing just gets ignored ayway. You could achieve a similar result by emitting a `logging.warning` message. Or... add some actual unit tests, rather than make me have to code around the lack of unit tests!

@thehrh : I get it, I've done this in the past, too, to make it more visible. We could add this in about a thousand places throughout PISA, ... :(

Overall, my philosophy is: We need lots more unit testing in PISA. We need people to feel unhappy about their own code until they have written good unit tests for it. BUT we also want Github CI tool to help us out, and it only helps us if a red X means someone broke something vs a green checkmark means they haven't broken anything (that was tested to work previously).

Any behavior besides that:, anything with any more subtlety than X vs Check, any requirements to read output, and the Github CI tool is useless (or not _useless_ per se, but it just will be _ignored_ by nearly everyone).

Reply:
Ok, I'm convinced. The side effects of purposely breaking the testing script would be pretty severe (given our current unit-testing status).